README.md

```diff
@@ -117,4 +117,43 @@ new HtmlWebPackPlugin({
   title: ...
 ```
 
-A good idea would be to conditionally change `template` path based on external config or an env variable.
\ No newline at end of file
+A good idea would be to conditionally change `template` path based on external config or an env variable.
+
+## Config
+
+### TCR (Optional)
+
+Tokens are dynamically loaded from the contract, but it might not be desirable to display everything in the interface.
```

Comment:
https://github.com/gnosis/dex-react/pull/983#discussion_r419324865

> Is not only not desirable, it's dangerous

> Maybe we should warn that Gnosis Protocol is a permissionless protocol, anyone can list any token, so if the tokens show up automatically, it can be not only dangerous for an user for having access to a potentially scammy token, but also it can be a token that is not legal under the jurisdictions that affect the deployed web.

> Probably we should make a short warning and check that one with Silke, so we encourage anyone using the web to maintain his own list complying with the different jurisdiction.

Reply:
Alright, I'll wait for a position from Silke before moving forward.